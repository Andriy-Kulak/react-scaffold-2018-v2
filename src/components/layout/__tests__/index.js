/* eslint-env node, jest */

import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Adapter from 'enzyme-adapter-react-16';

import Layout from '../index';

Enzyme.configure({ adapter: new Adapter() });

describe('>>> Layout Standard Render Component ~~ Snapshot ~~', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(<Layout><div>Test</div></Layout>);
    expect(toJson(renderedValue)).toMatchSnapshot();
  });
});