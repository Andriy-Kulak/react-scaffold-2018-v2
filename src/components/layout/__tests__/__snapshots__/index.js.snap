// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Layout Standard Render Component ~~ Snapshot ~~ +++capturing Snapshot with 1`] = `
<div>
  <Nav />
  <styled.div>
    <div>
      Test
    </div>
  </styled.div>
</div>
`;
